src/TetheredPopup.jsx

```diff
@@ -216,6 +216,7 @@ module.exports = React.createClass({
           anim.className = anim.className.replace(/ ?rw-popup-animating/g, '')
           anim.style.overflofw = 'visible';
 
+          contentEl.focus();
```

Comment:
Added this line to fix this bug: when I open a tethered popup list that's scrollable and scroll to the bottom, clicking on a list item causes the popup to scroll to the top instead of selecting the item. I think this was because it was not focused initially.